card-app/src/getUsers.js

```diff
@@ -0,0 +1,19 @@
+import { UserCard } from "./UserCard";
```

Comment:
You shouldn't need to import this here - this function will get used in the `UserCard` Theoretically, this function could be used in other spots of your app too, it's really flexible!